src/common/dns_utils.cpp

```diff
@@ -36,13 +36,21 @@
 #include <boost/filesystem/fstream.hpp>
 #include <boost/thread/mutex.hpp>
 #include <boost/thread/thread.hpp>
+#include <boost/algorithm/string/join.hpp>
 using namespace epee;
 namespace bf = boost::filesystem;
 
 #undef MONERO_DEFAULT_LOG_CATEGORY
 #define MONERO_DEFAULT_LOG_CATEGORY "net.dns"
 
-#define DEFAULT_DNS_PUBLIC_ADDR "8.8.4.4"
+static const char *DEFAULT_DNS_PUBLIC_ADDR[] =
+{
+  "194.150.168.168",    // CCC (Germany)
+  "81.3.27.54",         // Lightning Wire Labs (Germany)
+  "31.3.135.232",       // OpenNIC (Switzerland)
+  "80.67.169.40",       // FDN (France)
+  "209.58.179.186",     // Cyberghost (Singapore)
+};
```

Comment:
Can we also add `217.31.205.50` (nic.cz)? They also support DNSSEC.

Reply:
I can ping that host, but "dig -t TXT +dnssec @217.31.205.50 moneropulse.se" wedges.

Reply:
Oh my, I see now. Ok, nevermind.